starter-code/routes/books.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const router  = express.Router();
+const Book = require('../models/Book.model');
+
+/* GET books page */
+router.get('/books', (req, res, next) => {
+  Book
+    .find()
+    .then(books => {
+      res.render('books/index', { books });
+    })
+    .catch(err => console.error('Error getting the books', err));
+});
+
+/* GET new book form page */
+router.get('/books/new', (req, res, next) => {
+  res.render('books/new');
+});
+
+/* POST new book */
+router.post('/books', (req, res, next) => {

```

Comment:
is good practice to give the same url to both the get and post route in these cases. so /books/new